### PR TITLE
Add an uninstallation page.

### PR DESCRIPTION
**Pull request by msramek**
When chrome-extension://<id>/uninstall.html is opened, it will uninstall the extension. This can be used as an uninstallation link from a Help Center article.

### CODE REVIEW

File: chrome/uninstall.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="uninstall.js"></script>
+    <title></title>
+  </head>
+  <body>
+    <button id="uninstall">Uninstall</button>
```
Comment (mikewest) [inline chrome/uninstall.html]: Can you add the extension's name here? "Uninstall" isn't informative enough, especially given that the omnibox will display an opaque key's hash.

File: chrome/uninstall.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="uninstall.js"></script>
+    <title></title>
```
Comment (mikewest) [inline chrome/uninstall.html]: :-1: Give the page a title. Pages like titles!

File: chrome/uninstall.html (hunk 1 above)
Comment (msramek) [inline chrome/uninstall.html]: Sorry, I meant this just as a placeholder. I wanted to wait with i18n boilerplate code until we know which strings exactly we want to have. Nevertheless, I added it now :)

File: chrome/uninstall.html (hunk 2 above)
Comment (msramek) [inline chrome/uninstall.html]: I know! Ditto.

File: chrome/uninstall.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+/**
+ * @fileoverview This file is linked to uninstall.html, a page that can be
+ * pointed to from a Help Center article and will allow the user to uninstall
+ * the extension.
+ */
+
+
+window.addEventListener("load", function() {
+  var title = chrome.i18n.getMessage("extension_name");
```
Comment (mikewest) [inline chrome/uninstall.js]: Nit: Would you mind renaming `title` to `extensionName` or something similar?

**Comment by mikewest** [comment]
LGTM % nit.

File: chrome/uninstall.js (hunk 3 above)
Comment (msramek) [inline chrome/uninstall.js]: Done.

**Pull request merged**